pkg/cmd/factory/factory_impl.go

```diff
@@ -224,6 +225,15 @@ func (f *factoryImpl) findServerAddr() (string, error) {
 	}
 
 	ctx := cmdconetxt.GetDefaultContext()
+	err = etcdClient.CheckMultipleCDCClusterExist(ctx)
+	if err != nil {
+		if err == cerror.ErrMultipleCDCClustersExist {
+			return "", errors.New("You are using multiple TiCDC clusters to replica this " +
```

Comment:
Please just print the warning, instead of returning an error containing the long message.